stellarphot/utils/magnitude_system_transforms.py

```diff
@@ -229,20 +322,58 @@ def transform_apass_bands(table):
     # Putting this here to avoid a circular import
     from .magnitude_transforms import filter_transform
 
-    table["mag_RC"] = filter_transform(
-        table,
-        output_filter="R",
+    # Set up for input to Jester transform
+    use_columns = dict(
         g="mag_SG",
         r="mag_SR",
         i="mag_SI",
+    )
+
+    # The Jester transformation really for transformation from the ugriz on the
+    # 2.5m SDSS telescope to the Johnson-Cousins system.
+    #
+    # APASS is using u'g'r'i'z' (well, only g'r'i' in APASS) and so I *think*
+    # that the right thing to do is to transform the APASS g'r'i' to the SDSS DR7
```

Comment:
Will this add a significant amount of uncertainty to the transformed magnitudes if we are doing a double transform? If I am understanding this correctly.

Reply:
Not sure what the residual is in each of the fits and no one seems to express the coefficients as value ± error so we can't really propagate any transform errors.

Right now I *think* that using Jester only to change APASS r'i' to Rc and Ic is wrong. Jester's paper clearly states that he redid an earlier set of transforms to be *on the SDSS system*, not the USNO system, (aka. the "primed" system).